templates/assets/js/main.js

```diff
@@ -0,0 +1,82 @@
+const right_edge = '<div class="col-md-1 d-flex" id="right_edge" style="padding: 0;margin: 0;padding-top: 15px;padding-bottom: 15px;opacity: 1;">' +
+		'<div class="w-100 rounded-right" style="background-color: rgba(0,0,0,0.7);padding-top: 0;padding-bottom: 0;"></div>' +
+		'</div>';
+
+function ajaxCall(url, data, method, successCallback, errorCallback) {
+	$.ajax({
+		method: method,
+		dataType: "json",
+		url: url,
+		data: data,
+		success: successCallback,
+		error: errorCallback
+	});
+}
+
+function signupBtn() {
+	ajaxCall(
+		'signup',
+		{
+			'username': $("#main_form_email").val(),
+			'password': $("#main_form_password").val()
+		}, "POST", 
+		function(data) {
+			window.location.replace("/");
+		}, function(data) {
+			console.log(data);
+			$("#error_message").text(data).delay(5000).fadeOut(400);
+		}
+	);
+}
+
+function signinBtn() {
```

Comment:
Just a suggestion, this is not a button therefore naming it signinBtn for my opinion is wrong and missleading.
I think all functions names should be named like actions.
For example, this function should be named "signin".

Reply:
@DoRTaL94 same here